charts/opentelemetry-collector/templates/NOTES.txt

```diff
@@ -13,3 +13,7 @@
 {{- if and (not (eq .Values.mode "daemonset")) (not (eq .Values.mode "deployment")) }}
 {{ fail "[ERROR] 'mode' must be set. See https://github.com/open-telemetry/opentelemetry-helm-charts/blob/main/charts/opentelemetry-collector/UPGRADING.md for instructions." }}
 {{ end }}
+
+{{- if and (not .Values.configMap.create) (or (eq (toString .Values.extraVolumes) "<nil>") (eq (toString .Values.extraVolumeMounts) "<nil>") ) }}
```

Comment:
I don't think we want to strictly enforce `.Values.extraVolumes` or `.Values.extraVolumeMounts` be set when `.Values.configMap.create` is false.  There are situations where users may build their own collector image with the config included and then not need a configmap at all.  I do think we should enforce that `.Values.config` be set to `null` when `.Values.configMap.create` is true.

Reply:
when `.Values.configMap.create` is true,  Whatever the value of the ` `.Values.config``, it does not work, I think we can give an warning msg.